.github/workflows/cd.yaml

```diff
@@ -1,6 +1,11 @@
 name: CD
 
 on:
+  push:
```

Comment:
What about cloud-cd.yaml ? That should also be included to make sure that cloud deployment works fine on main(edc mvd upstream), correct ?

Reply:
Thanks, implemented